openedx/core/djangoapps/ace_common/template_context.py

```diff
@@ -16,6 +16,7 @@ def get_base_template_context(site):
     """
     # When on LMS and a dashboard is available, use that as the dashboard url.
     # Otherwise, use the home url instead.
+    default_logo_url = 'https://edx-cdn.org/v3/default/logo.png'
```

Comment:
Nit: Consider putting the default values for each of the Logo settings in `common.py` rather than within the application layer code.

Reply:
good point, thank you.